eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/EnactSpec.hs

```diff
@@ -429,7 +429,84 @@ actionPrioritySpec ::
   , InjectRuleFailure "LEDGER" ConwayGovPredFailure era
   ) =>
   SpecWith (ImpInit (LedgerSpec era))
-actionPrioritySpec =
+actionPrioritySpec = do
+  describe "Competing proposals - separated out for conformance mismatch" $
```

Comment:
As I mentioned in our call, let's avoid moving tests around for the sake of this conformance and instead of marking a whole section as non-ready for conformance add `disableImpInitExpectLedgerRuleConformance` to each individual `it` with a note to a github issue